### DISCUSSION TITLE
coc#pum#visible() ? coc#pum#prev(0) : "\"   2   coc#pum#visible() ? coc#pum#next(0) : "\"

### DISCUSSION THREAD

**Discussion by alprofmahmoud** [neoclide/coc.nvim#4852] [Q&A]
![Screenshot_20240103_150619_Termux](https://github.com/neoclide/coc.nvim/assets/35482927/27c16350-1413-4aa9-ae87-765d18059856)

It is possible to help solve this problem when using coc.vim
When the Up and Down buttons are pressed in Insert mode
What is the solution to this problem?

**Comment by fannheyward** [comment]
How do you setup your mappings?
